### PR TITLE
Corgi

### PR DESCRIPTION
**Pull request by MiloRK**
Adds the Tricolor Corgi

### CODE REVIEW

**Comment by MiloRK** [comment]
Sorry for the delay. I took me a while to figure out what I was doing wrong with GitHub.

File: Defs/ThingDefs/Races_Animal_Pet_Corgi.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        <meleeDamageBaseAmount>10</meleeDamageBaseAmount>
+        <meleeDamageDef>Bite</meleeDamageDef>
+        <linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+      </li>
+    </verbs>
+    <race>
+      <body>QuadrupedAnimalWithPawsAndTail</body>
+      <petness>1</petness>
+      <herdAnimal>true</herdAnimal>
+      <baseBodySize>1.0</baseBodySize>
```
Comment (Skullywag) [inline Defs/ThingDefs/Races_Animal_Pet_Corgi.xml]: update body size to 0.5

File: Defs/ThingDefs/Races_Animal_Pet_Corgi.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      <litterSizeCurve>
+        <points>
+          <li>(0.5, 0)</li>
+          <li>(1, 1)</li>
+          <li>(2, 1.7)</li>
+          <li>(3, 0.7)</li>
+          <li>(3.5, 0)</li>
+        </points>
+      </litterSizeCurve>
+      <lifeExpectancy>15</lifeExpectancy>
```
Comment (Skullywag) [inline Defs/ThingDefs/Races_Animal_Pet_Corgi.xml]: google says max age is 14 and ive been following that for all other dogs

**Comment by MiloRK** [comment]
I think the corgi is ready to go :)

**Pull request merged**